src/applications/combined-debt-portal/combined/containers/CombinedPortalApp.jsx

```diff
@@ -36,7 +40,7 @@ const CombinedPortalApp = ({ children }) => {
   useEffect(
     () => {
       if (userLoggedIn) {
-        fetchDebtLetters(dispatch);
+        fetchDebtLetters(dispatch, debtLettersActive);
```

Comment:
@kjsuarez I'd just add `debtLettersActive` do the dependency array on line 47, otherwise looks good!

Reply:
On it